posts/inside-rust/2023-08-21-clippy-perf-project.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Making Clippy blazingly fast by 2024"
```

Comment:
When I first saw this title I assumed end of 2023/January 2024 whereas it's really an end of 2024/Jan 2025. 

Perhaps that's just an odd interpretation from my end, but if you think others may potentially initially assume the same, then it may be worth expanding this a tad, e.g.:

```suggestion
title: "Making Clippy blazingly fast by the end of 2024"
```

Reply:
Yeah, I can see how it could be interpreted like that, definitely a confusing title.